tests/test_rewrite_paragraph.py

```
@@ -0,0 +1,702 @@
+        ref = refs[0].split("|")[0]
+
+        doc.rewrite_paragraph(ref, "The cat and the dog and a bird")
+
+        vis = doc.get_visible_text()
+        assert "The cat and the dog and a bird" in vis
+
+        revisions = doc.list_revisions()
+        del_texts = [r.text for r in revisions if r.type == "deletion"]
+        ins_texts = [r.text for r in revisions if r.type == "insertion"]
+        assert any("the" in t.lower() for t in del_texts)
+        assert any("a" == t.strip() for t in ins_texts)
```

Ruff (0.15.7)

[warning] Yoda condition detected

Rewrite as `t.strip() == "a"`

(SIM300)